source/code/shared/tools/scx_ssl_config/scx_ssl_config.cpp

```diff
@@ -461,6 +462,15 @@ static int DoGenerate(const wstring & targetPath, int startDays, int endDays,
         if (0 != rc) {
             throw SCXCoreLib::SCXErrnoFileException(L"chmod", keyPath.Get(), errno, SCXSRCLOCATION);
         }
+
+        struct passwd *pwd=NULL;
+        if((pwd = getpwnam("omi")) != NULL)
+        {
+            rc = chown(sKeyFile.c_str(),pwd->pw_uid,pwd->pw_gid);
+            if (0 != rc){
+                throw SCXCoreLib::SCXErrnoFileException(L"chown", keyPath.Get(), errno, SCXSRCLOCATION);
+            }
+        }
```

Comment:
Don't you want to log an error if `getpwnam` fails? In fact, after making your changes, please (without a kit installed, so without the `omi` account existing) run the program and insure a meaningful error is printed.

Reply:
Modifeid the code to log error if getpwnam fails. I think logs content will be fine as I have followed the convention used for other syscalls.

Reply:
Verified the logs and it looks fine.